app/src/main/java/com/imashnake/animite/ui/elements/home/SearchBar.kt

```diff
@@ -75,16 +80,7 @@ fun ExpandedSearchBar(viewModel: SearchViewModel = viewModel()) {
                     searchAnime(input)
                 }
             },
-            label = {
-                Text(
-                    text = "Search",
-                    color = Text.copy(alpha = 0.5F),
-                    fontSize = 12.sp,
-                    maxLines = 1,
-                    fontFamily = manropeFamily,
-                    fontWeight = FontWeight.Medium
-                )
-            },
+            placeholder = { Text(text = "Search", fontSize = 16.sp, color = Text) },
```

Comment:
We should also let the other attributes persist, but I can resolve this in the PR where I add the action for the clear `trailingIcon`.